src/ArgentPonyWarcraftClient/Models/ProfileApi/CharacterSoulbinds/ConduitSocket.cs

```diff
@@ -5,7 +5,7 @@ namespace ArgentPonyWarcraftClient
     /// <summary>
     /// Rank details for a conduit socket.
     /// </summary>
-    public class ConduitRank
+    public class ConduitSocket
```

Comment:
Renamed to better align with the model's intent and to remove conflict with the `ConduitRank` model that's part of the Covenant API. Open to other suggestions for a name on this, but it felt like an appropriate name for the context its in.

Reply:
That's fine.  There were several nested types related to this, and naming them well wasn't easy.